docker/iml-gui.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rust:1.41 as builder
+WORKDIR /build
+COPY . .
+RUN cd /build/iml-gui && \
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+    curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+    apt-get install -y yarn nodejs && \
+    rustup target add wasm32-unknown-unknown && \
+    cargo install wasm-pack && \
+    yarn install && \
+    yarn build:release;
+
+FROM ubuntu
```

Comment:
Can this not be centos?

Reply:
It doesn't matter, we don't use it in production. The files are copied out of this container and into the nginx one.

Reply:
Ok, just wanted to make sure we don't introduce a new image if we can avoid it.